bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/model/Evaluation.kt

```diff
@@ -0,0 +1,15 @@
+package io.bucketeer.sdk.android.internal.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Evaluation(
+  val id: String,
```

Comment:
Everything is non-null for now, will be updated once swagger provides nullability information.